### PR TITLE
TRIVIAL: Fix backend_type warning message

### PR DESCRIPTION
**Pull request by caohhung**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 require 'net/ssh'
 require 'rspec/its'
 require './cfg/cfg_helper'
+set :backend, :exec
```
Comment (msurovcak) [inline spec/spec_helper.rb]: This will effectively disable `:ssh` backend.

What are you trying to fix?

Comment (martin-ducar-gd) [inline spec/spec_helper.rb]: Try to look at https://github.com/gooddata/puppet/blob/develop/modules/serverspec/manifests/core.pp#L43 it should be set but for some reason its not used or failing

Comment (martin-ducar-gd) [inline spec/spec_helper.rb]: It looks like some code in this repo removed setting backed based on that ENV variable

**Comment by ytsarev** [comment]
fixed by https://github.com/gooddata/serverspec-core/pull/50

**Pull request closed without merging**